CONTRIBUTING.md

```diff
@@ -1,501 +1,108 @@
-
-The gno project currently favors a mono-repo structure, as it’s easier to manage contributions and keep everyone
-aligned. In the future, this may change, but in the meantime the majority of gno resources and source code will be
-centralized here.
-
-- `examples` - contains the example `.gno` realms and packages. This is the central point for adding user-defined realms
-  and packages.
-- `gno.land` - contains the base source code for bootstrapping the Gnoland node,
-  using `tm2` and `gnovm`.
-- `gnovm` - contains the implementation of the Gno programming language and its
-  Virtual Machine, together with their standard libraries and tests.
-- `tm2` - contains a fork of the [Tendermint consensus engine](https://docs.tendermint.com/v0.34/introduction/what-is-tendermint.html) with different expectations.
-
-## How do I?
-
-### How do I submit changes?
-
-Pull Requests serve primarily as an addition to the codebase, but also as a central point of discussion for potential features.
-Instead of opening issues, contributors are encouraged to open PRs with their suggested changes, however small, in order to
-get feedback from the community on their work.
-
-The key to successfully submitting upstream changes is providing the adequate context, with the correct implementation,
-of course.
-
-To let the maintainers know about _what_ your PR does, and _why_ it does it, you should take a second to fill out the PR
-template provided on the repo.
-
-Once someone leaves a PR review (with open comments / discussions), it is on the PR _creator_ to do their best in trying
-to resolve all comments.
-The PR _reviewer_ is in charge of resolving each conversation (hitting the `Resolve Conversation` button), after the PR
-_creator_ implements or discusses
-the requested changes.
-
-If the PR _creator_ pushed code as a result of some conversation, they should link the commit hash in the relevant
-comment reply, ex:
-
-```Markdown
-Fixed in:
-
-[0a0577c](https://github.com/gnolang/gno/commit/0a0577ccdeb951a6621d6fbe1c04ac4e64a529c1)
-```
-
-#### A Word on Rebasing
-
-Avoid rebasing after you open your PRs to reviews. Instead, add more commits to your PR.
-It's OK to do force pushes if PR was never opened for reviews before.
+### Git Commit Messages
 
-A reviewer may like to see a linear commit history while reviewing. If you tend to force push from an older commit,
-a reviewer might lose track in your recent changes and will have to start reviewing from scratch.
+Follow [Conventional Commits](https://www.conventionalcommits.org/):
+- `feat(scope): add new feature`
+- `fix(scope): fix issue`
+- `docs(scope): update documentation`
 
-Don't worry about adding too many commits. The commits are squashed into a single commit while merging (if needed).
+### Rebasing
 
-### How do I report a bug?
+Avoid rebasing after opening your PR for review. Maintainers handle the final
+squash/merge. Add new commits to address feedback instead of force-pushing.
```

Comment:
```suggestion
squash/merge. Add new commits to address feedback instead of force-pushing.

Using merge commits instead of rebase allows reviewers
to better review changes only since their last review.
To disable rebase when using `git pull` on the repository, run:

	git config pull.rebase false

This will be disabled only for the git repository you're currently on.
```